app/controllers/api/framework_assessments_controller.rb

```diff
@@ -16,12 +16,12 @@ class FrameworkAssessmentsController < ApiController
     ].freeze
 
     def create
-      assessment = assessment_class.save_with_responses!(
+      new_assessment = assessment_class.save_with_responses!(
         version: new_assessment_params.dig(:attributes, :version),
         move_id: new_assessment_params.dig(:relationships, :move, :data, :id),
       )
 
-      render_assessment(assessment, :created)
+      render_assessment(assessment(new_assessment.id), :created)
```

Comment:
we can't include resources on a single resource, which is forcing us to find it again, happy for any alternative ideas on how to do this

Reply:
Hmm... interesting. So I think I follow the code here - `save_with_responses` is creating a new framework instance, then persisting it with all of the associated framework responses etc. So at this point you have the corresponding record object, but can't then call `.includes()` on it.

If you think about it, at that point all of the object relationships should already have been loaded into memory, so there's no need to retrieve it from the database again. I suspect that this change is actually performing more work now than before 🤔

Reply:
So through testing unfortunately it was actually ignoring all the loaded records and fetching them again when loading the serializer. But because we didn't include the resources the n+1 queries were substantial. By adding only a find with the includes after the creation we avoid all the many queries

Reply:
Explored this in person - @cwrw is correct regarding the strange `includes` behaviour for a record that's already (in theory) loaded. This fix means one additional simple query to fetch the assessment again, but that mitigates a *lot* of potential further N+1 queries so is an acceptable approach - I can't think of anything better than this that would work either. All good 👍